src/hash_table.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+
+class HashTable(object):
+    """Hash Table Class."""
+
+    def __init__(self, value):
+        """Initialize hashtable of size value."""
+        self.table = []
+        for i in range(value):
+            self.table.append([])
+
+    def _hash(self, key):
+        """Take a key and hash it. Return and integer."""
+        hash_value = 0
+        for character in key:
+            hash_value += ord(character)
+        return hash_value % len(self.table)
+
+    def set(self, key, value):
+        """Set the value using the key in the hash table."""
+        self.table[self._hash(key)].append((key, value))
```

Comment:
Also, of course Python throws a TypeError when you try to call `ord()` on a non-string character. However, the TypeError thrown doesn't imply that the key itself absolutely must be a string. Maybe I'm being nitpicky?